src/pharmpy/tools/external/nonmem/results.py

```diff
@@ -465,12 +465,12 @@ def _parse_ofv(ext_tables: NONMEMTableFile, subproblem: Optional[int]):
     iteration = []
     ofv = []
     final_table = None
-    for i, table in enumerate(ext_tables.tables, start=1):
+    for table_number, table in enumerate(ext_tables.tables, start=1):
```

Comment:
`i` is generally accepted to be a universal counting variable and what is being counted is clear from the  `for`. This change makes it harder later in the code to understand that `table_number` is the counting variable. That being said it makes it easier to see what the counting variable is. To me it is a net zero change.

Reply:
Better to be explicit than implicit?